checker/check-request-body-required-value-updated_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/tufin/oasdiff/diff"
 )
 
-// BC: changing request's body to required is breaking
+// CL: changing request's body to required is breaking
```

Comment:
is it intentional to change this from BreakingChange scenario to ChangeLog scenario?

Reply:
BC entries are tracked in checker_breaking.go and checker_not_breaking.go 
- currently using check-<checker>_test.go files to track all changelog entries that are captured there. 
- open to discuss a target approach to capturing those entries and open a PR for that specifically

Reply:
thanks for clarifications